integration/inspektor-gadget/audit_seccomp_test.go

```diff
@@ -62,20 +57,54 @@ data:
         ]
     }
 ---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: installer-sa
+  namespace: %[1]s
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: installer-cluster-role-%[1]s
+rules:
+  - apiGroups: ["security.openshift.io"]
+    # It is necessary to use the 'privileged' security context constraints to be
+    # able mount host directories as volumes, use the host networking, among others.
+    # This will be used only when running on OpenShift:
+    # https://docs.openshift.com/container-platform/4.9/authentication/managing-security-context-constraints.html#default-sccs_configuring-internal-oauth
```

Comment:
Should we use a more recent version to avoid the warning on the webpage?
```suggestion
    # https://docs.openshift.com/container-platform/4.14/authentication/managing-security-context-constraints.html#default-sccs_configuring-internal-oauth
```